### PR TITLE
add ALTAIR_FORK_EPOCH for mainnet configuration

### PR DESCRIPTION
**Pull request by djrtwo**
***Assuming no issues, this value for `ALTAIR_FORK_EPOCH` will be released in a `v1.1.0` on Monday, Sept 27.***

Rationale -- 
* reasonable-ish time for most contributors (sorry California)
* `(ALTAIR_FORK_EPOCH * SLOTS_PER_EPOCH) % SLOTS_PER_HISTORICAL_ROOT == 0` -- thus types across forks for the batching are cleanly separated

### CODE REVIEW

File: configs/mainnet.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ GENESIS_DELAY: 604800
 
 # Altair
 ALTAIR_FORK_VERSION: 0x01000000
-ALTAIR_FORK_EPOCH: 18446744073709551615
+ALTAIR_FORK_EPOCH: 74240  # Oct 27, 2021, 10:56:23pm UTC
```
Comment (terencechain) [inline configs/mainnet.yaml:32]: I think this is am

```suggestion
ALTAIR_FORK_EPOCH: 74240  # Oct 27, 2021, 10:56:23am UTC
```

Comment (djrtwo) [reply configs/mainnet.yaml:32]: indeed it is! ha thank you!

**Review by terencechain — APPROVED** [review]

**Review by hwwhww — APPROVED** [review]
🟢

File: configs/mainnet.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ GENESIS_DELAY: 604800
 
 # Altair
 ALTAIR_FORK_VERSION: 0x01000000
-ALTAIR_FORK_EPOCH: 18446744073709551615
+ALTAIR_FORK_EPOCH: 74240  # Oct 27, 2021, 10:56:23am UTC
```
Comment (protolambda) [inline configs/mainnet.yaml:32]: LGTM. Sanity check: `1606824023+(74240*32*12)=1635332183`, `Wed Oct 27 2021 10:56:23 GMT+0000`.
And extra: `(74240*32)%(2**13)=0` (fresh block/state roots batch)

**Review by protolambda — APPROVED** [review]

**Review by michaelsproul — APPROVED** [review]
Good for us! Should be included in Lighthouse v1.6.0 later this week

**Review by dapplion — APPROVED** [review]
Lodestar :+1:

**Review by jley81 — APPROVED** [review]

**Comment by favealo** [comment]
👍

**Pull request merged**